### PR TITLE
[TACHYON-1240] Improved logging message formats in tachyon-underfs-* modules

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1240

### CODE REVIEW

**Comment by aaudiber** [comment]
Thanks for the improvement! LGTM.

File: underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public HdfsUnderFileSystem(String fsDefaultName, TachyonConf tachyonConf, Object
     try {
       mFs = path.getFileSystem(tConf);
     } catch (IOException e) {
-      LOG.error("Exception thrown when trying to get FileSystem for " + mUfsPrefix, e);
```
Comment (gpang) [inline underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java]: Will this still work for the syntax where the last argument is an exception?

**Comment by gpang** [comment]
I just had a minor question about the new syntax and exception arguments. Thanks!

**Comment by HorizonNet** [comment]
@gpang That was a hint from @apc999 in another [PR](https://github.com/amplab/tachyon/pull/2011) with a link to the [documentation](http://www.slf4j.org/faq.html#paramException) of SLF4J. The documentation states "...supports parametrization in the presence of an exception, assuming the exception is the last parameter". This will work with version 1.6.0. Tachyon uses version 1.7.2.

File: underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java (hunk 1 above)
Comment (aaudiber) [inline underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java]: If the format string specifies `n` parameters and the arguments are `n` parameters followed by an exception, the exception will be printed with stack trace on the line following the log message.

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**